### DISCUSSION TITLE
consiglio per citofono

### DISCUSSION THREAD

**Discussion by SkunkGuru** [indomus/forum#2694] [GENERALE - Discussioni generiche]
Salve a tutti !
Esiste uno shelly o similare con doppio contatto pulito ? 
vorrei metterlo dove ho poco spazio e due shelly1 proprio non ci stanno 😞  (ossia dentro il citofono per aprire 2 portoni distinti)
non avendo avuto successo col Ring volevo riuscire almeno a domotizzare l'apertura di portone e cancello

**Comment by indomus** [comment]
https://indomus.it/componenti/itead-sonoff-4ch-4ch-pro-r3/

**Reply by SkunkGuru** [reply]
buono a sapersi, peccato che sia grandino, mi sa più di 2 shelly affiancati

**Comment by barbatrukko** [comment]
Dentro un citofono è difficile mettere 2 dispositivi. Ma il tuo citofono ha 2 pulsanti?
Potresti usare questo per rilevare la pressione dei due pulsanti e poi da home assistant o da app gestire i segnali
[Shelly plus i4](https://www.shellyitalia.com/shelly-plus-i4/?gad_source=1&gclid=CjwKCAiA3aeqBhBzEiwAxFiOBgPuB_-CHIzPWkFLsidAsUskwCKztk-asMo88NU4r4DIO3NPsyxuzRoCcE0QAvD_BwE)
[Shelly plus i4 DC](https://www.shellyitalia.com/shelly-plus-i4-dc/)
 c'è alimentato in AC e in DC
In pratica nel citofono metti solo questo, e dove hai spazio due shelly/sonoff qualsiasi per gestire le aperture separate.
In pratica separi l'azione di reperire il comando da quella di attuare il comando.

**Reply by SkunkGuru** [reply]
Si, il citofono ha 2 pulsanti, uno apre il portone vero e proprio, l'altro il cancellino pedonale esterno, e ovviamente per dare l'accesso vanno sempre premuti entrambi; avevo provato a collegare assieme i due comandi ma poi scattavano sempre anche dagli altri condomini 😛  grazie per l'idea

**Comment by SkunkGuru** [comment]
aggiornamento + problema

alla fine ho installato il 4ch pro r3 dietro un mobiletto vicino, usando un cavo per portare "fuori" dal citofono i cavetti che mi servivano; ho collegato i due ingressi del sonoff e tutto funziona perfettamente. Ho integrato in HA tramite l'integrazione "Sonoff LAN"


se non fosse che... mi sono accorto che, in orario senza logica, pare che il sonoff attivi tutti i canali (compresi quelli non collegati) per conto suo, senza che venga richiesto da automazioni o altro

esempio stanotte, questo ingresso 1 (che non uso, per cui non ci sono automazioni o script o nulla che lo citi) sembra essersi attivato
![image](https://github.com/indomus/forum/assets/15073199/b0c461fe-ab5a-4a61-94bf-96dbda08f10d)

non c'è nulla di configurato neanche nella app (ewelink) che ho usato solamente per la prima configurazione e per impostare l'inching; sempre tramite app ho impostato che alla riaccensione del dispositivo lo stato sia "spento" per cui escluderei riavvi o altro; il dispositivo non è condiviso via app (che ho solo io comunque)

nei registri della app appare questo messaggio, sembra effettivamente un comando di "accendi tutto", ma non capisco da dove possa arrivare
![image](https://github.com/indomus/forum/assets/15073199/ce62dc0b-80b3-47ed-ac90-165d5275b694)

spero qualcuno abbia qualche indicazione da darmi 😢

**Reply by SkunkGuru** [reply]
anche oggi un'attivazione "spontanea"... quella delle 13.14 è voluta, attivata tramite HA (e infatti riguarda i canali che ho usato, 2 e 4), ma quelle CHAll delle 14.43 non so da cosa siano state attivate... e poi non capisco il "local button" che avrebbe attivato i 4 ingressi... non c'è un bottone unico, ma uno per canale, per cui... ? dite potrebbe essere difettoso ?

![image](https://github.com/indomus/forum/assets/15073199/8c766940-51ba-40a6-816d-69d56e2a1c04)

**Comment by barbatrukko** [comment]
Veramente strano...
Non mi è mai capitato: quindi ti triggera eventi su canali non usati?

**Reply by SkunkGuru** [reply]
esatto, si attiva automaticamente su tutti i canali, anche con tutto scollegato

dopo 7/8 email con il supporto sonoff adesso stanno ipotizzando un disturbo RF con altri apparati (che non ho) per cui il loro consiglio è aprirlo e rimuovere l'antenna (!!!!) tagliando un componente

penso che a questo punto farò il reso ad amazon...
